include/string.h

```diff
@@ -24,10 +24,10 @@ extern "C" {
 
 // Returns a pointer to dest.
 // Copies n bytes from src to the memory region dest points to.
-void *memcpy (void *__restrict dest : itype(__restrict _Array_ptr<void>) byte_count(n),
-              const void *__restrict src : itype(__restrict _Array_ptr<const void>) byte_count(n),
-              size_t n)
-  : itype(_Array_ptr<void>) byte_count(n);
+void *memcpy (void *__restrict dest : itype(__restrict _Array_ptr<void>) byte_count(arg_n),
```

Comment:
It might be better to note rename parameters in header files.   The name could be tied into documentation etc.   It is OK for the declaration and definition of a function to have different parameter names.   It typechecks fine.  C cares about the argument position not the name.   We're introducing `arg_n` because of  Checked C.     I am wondering if clang issues a warning if the definition and declaration have different parameter names.

Reply:
True. Thank you for your review and for pointing this out. Clang does not issue a warning if a function definition and declaration have different names for parameters. I will revert all the parameter renaming that I have done in the header files.